src/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -277,6 +277,37 @@ export function getUiSettingDefaults() {
 }`,
       description: 'The timefilter\'s default refresh interval'
     },
+    'timepicker:quickRanges': {
```

Comment:
Is there a way to add a `verify` method to ensure the time ranges are valid? It would be nice to get an error message and not update the quickRanges on the advanced setting page, then to get an error when selecting the invalid range via the timepicker.

Reply:
I don't think there's any sort of verification currently (beyond verifying that the value actually is JSON). I could probably add a custom directive and handling for this but it might add more complexity than it's worth... Plus, there are other JSON values in the advanced settings that we don't have any sort of verification either. What if I just added some notes to the docs about how to configure the value and linked to it in the description?

Reply:
sounds good

Reply:
@nreese 1eb0eee